src/main/scala/beam/agentsim/infrastructure/ChargingNetworkManager.scala

```diff
@@ -396,77 +395,21 @@ object ChargingNetworkManager {
   case class UnhandledVehicle(tick: Int, vehicleId: Id[BeamVehicle])
 
   final case class ChargingZone(
-    chargingZoneId: Int,
     tazId: Id[TAZ],
     parkingType: ParkingType,
     numChargers: Int,
     chargingPointType: ChargingPointType,
     pricingModel: PricingModel,
     managerId: Id[VehicleManager]
   ) {
-    val uniqueId: String = s"($managerId,$tazId,$chargingZoneId,$parkingType,$chargingPointType)"
+    val id: String = s"cs_${managerId}_${tazId}_${parkingType}_${chargingPointType}"
```

Comment:
it would be better to move key generation to one function and reuse this function in other places

Reply:
Done!